code/controllers/subsystem/persistent_data.dm

```diff
@@ -0,0 +1,58 @@
+/**
+  * # Persistent Data Subsystem
+  *
+  * Provides a centralised handler for persistent data reading and writing.
+  * The subsystem does not do any actual spawning itself, as this focuses on objects and mobs
+  * Should anything that is turf persistence related added in, that can be chucked into this SS
+  * Its quite a simple subsystem. For now, anyways.
+  *
+  */
+SUBSYSTEM_DEF(persistent_data)
+	name = "Persistent Data"
+	init_order = INIT_ORDER_PERSISTENCE // -95 | Loads after EVERYTHING else
+	flags = SS_NO_FIRE
+	/// List of atoms registered into the subsystem for persistent data storage. Can be anything at all
+	var/list/registered_atoms = list()
+
+/datum/controller/subsystem/persistent_data/Initialize()
+	// Load all the data of registered atoms
+	for(var/atom/A in registered_atoms)
+		A.PersistentLoad()
+	return ..()
+
+/datum/controller/subsystem/persistent_data/Shutdown()
+	// Save all the data of registered atoms
+	for(var/atom/A in registered_atoms)
+		A.PersistentSave()
+
+
+/**
+  * Proc to register an atom with SSpersistent_data
+  *
+  * This will add any provided atom to the list of registered atoms, and add it to the Initialization queue
+  * If the system has already initialized, it calls PersistentLoad() at that moment
+  *
+  * Arguments:
+  * * A - Atom to register
+  */
+/datum/controller/subsystem/persistent_data/proc/register(atom/A)
```

Comment:
Best to also make a unregister proc. Will do the same as you have now in other places. The -=. But it will be a bit cleaner than directly accessing the list

Reply:
I dont see the point since `register` was only added for loading things at runtime, that arent placed on the map. If you look at the original commit for this PR, using `register()` is cleaner than the if check each time

Reply:
It has no logic behind it yet. Yet it might get some logic behind it.
Also makes it more readable than removing it from a list. Will also remove the suggestion you can just add an atom to the list using += or |=

Reply:
I cant think of any logic at all that would be behind this. If an object has already been saved, it should be out of the list, since thats the only reason the list exists, for either invoking load on initialize, and for saving at round end.